### DISCUSSION TITLE
Edit extension only works with an ending slash, is this the intended behaviour?

### DISCUSSION THREAD

**Discussion by sorenpeter** [datenstrom/community#500] [Ask a question] [answered]
Hey,

Just found out that accessing the edit-feaute only works if the URL ends with a slash /
this does not work: myurl.dk/edit 
but this does: myurl.dk/edit/

**Comment by schulle4u** [comment, marked as answer]
Yes, this is the intended behaviour as described in the [extension documentation](https://github.com/datenstrom/yellow-extensions/tree/master/features/edit). You can of course point the edit extension directly to pages inside folders, e. g. https://website/edit/about/me

**Comment by sorenpeter** [comment]
ah sure. (note to self: rtfm)
